### DISCUSSION TITLE
osu!lazer not opening

### DISCUSSION THREAD

**Discussion by dacodes1** [ppy/osu#15902] [Q&A] [answered]
hi
my osu lazer is not op[ening or showing in the start bar
here are my logs
[performance.log](https://github.com/ppy/osu/files/7642341/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7642342/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7642343/updater.log)
[database.log](https://github.com/ppy/osu/files/7642344/database.log)
[network.log](https://github.com/ppy/osu/files/7642345/network.log)

**Comment by smoogipoo** [comment]
https://github.com/ppy/osu/issues/15836

**Reply by dacodes1** [reply]
> #15836

i dont use windows 11 tho?

**Comment by peppy** [comment, marked as answer]
This will be fixed next release, please wait for that release and then install manually.

**Reply by dacodes1** [reply]
plz dont kill me but.. how do you manually install?

**Reply by dacodes1** [reply]
does this mean like reinstall? since i dont wanna lose my played beatmaps

**Reply by peppy** [reply]
https://github.com/ppy/osu#running-osu run install.exe from here.

**Reply by dacodes1** [reply]
will all my beatmaps still be there?

**Reply by peppy** [reply]
yes.
